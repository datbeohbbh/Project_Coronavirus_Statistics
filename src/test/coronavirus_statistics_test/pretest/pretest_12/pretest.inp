21 26
? 5
? 5
q 1 7 139
q 1 2 11
? 4 2
q 1 13 238
? 4 2
q 1 10 257
q 2 10 46
q 1 11 453
q 1 19 149
q 2 13 218
? 7
q 1 15 33
q 1 9 236
q 1 1 396
? 4 6
? 4 4
q 2 1 34
? 6
? 4 1
? 7
? 7
? 6
? 4 6
? 6