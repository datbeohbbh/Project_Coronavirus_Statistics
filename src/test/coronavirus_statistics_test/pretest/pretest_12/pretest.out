Query ? 5 :
The number of safe cities is 21

Query ? 5 :
The number of safe cities is 21

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 139
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 139
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 6 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 211
-> Has recovered : 46
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 139
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 218 people

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 11
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 218 people

Query ? 4 6 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 211
-> Has recovered : 46
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 218 people

